docs/src/pages/guides/examples.md

```diff
@@ -5,7 +5,7 @@ title: Examples
 
 ## Minimal Configuration
 
-Here is a probe example with GET request to hit github.com:
+Creating a probe configuration is fairly simple. You only need a `probe` key type, a probe type such as an (http) `request` and the `url` to probe. The example below performs a GET probe request to github.com:
```

Comment:
I think it's better to avoid saying something is 'simple'. Better say:

```
At minimum, you only need to specify the URL you want to monitor. For example, to monitor github.com, you can use the following configuration:


```